docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
+telescopes, and calibration devices.
+The management, definition, derivation, verification, and validation of the simulation model is central to the functionality of simtools.
+
+Examples for model parameters are the atmospheric models (profile, transmission) or geomagnetic field characteristics for the site,
+description of optical and mechanical properties of the telescope, and detector plan, trigger, and readout settings for the cameras.
+
+For simplicity, the term parameter is used for values, (multi-dimensional) vectors of values
+(e.g., the mirror reflectivity vs wavelength and photon incident angle), functions, and algorithms
+(e.g., the telescope trigger algorithm).
+Parameters might be fixed in time (e.g., the number of mirrors on a certain telescope), or variable
+on different time scales (e.g., compare the degradation of the mirror reflectivity over several months
+with the nightly variability of the atmospheric parameters).
+
+The major components to handle the simulation model in simtools are:
+
+1. **Model parameter databases:** The simulation model is stored in [mongoDB databases](databases.md#databases).
+The {ref}`db_handler module <DBHANDLER>` provides reading and writing interfaces to the database.
+2. **Model parameter management:** The {ref}`model_parameters module <model_parameters>` provides interfaces to manage the simulation model parameters.
+3. A **model of an array and its elements** consists of the {ref}`SiteModel <site_model>` and several {ref}`TelescopeModel <telescope_model>` and {ref}`CalibrationModel <calibration_model>` instances.
+
+Simtools includes applications to write new model parameters to the databases and the export or import model parameters from sim_telarray configuration files.
```

Comment:
I prefer to list what capabilities our applications have and point to each one of that. It is too short here for a lot of unrelated detailed information.

Reply:
Added a "(see sections below)", as we are very detailed in the following sections.